src/pages/Search/SearchTypeMenu.tsx

```diff
@@ -83,36 +190,41 @@ function SearchTypeMenu({queryJSON}: SearchTypeMenuProps) {
                 activeItemIndex={activeItemIndex}
                 queryJSON={queryJSON}
                 title={title}
+                savedSearchesMenuItems={savedSearchesMenuItems()}
             />
         );
     }
 
     return (
-        <View style={[styles.pb4, styles.mh3, styles.mt3]}>
-            {typeMenuItems.map((item, index) => {
-                const onPress = singleExecution(() => {
-                    SearchActions.clearAllFilters();
-                    Navigation.navigate(item.route);
-                });
-
-                return (
-                    <MenuItem
-                        key={item.title}
-                        disabled={false}
-                        interactive
-                        title={item.title}
-                        icon={item.icon}
-                        iconWidth={variables.iconSizeNormal}
-                        iconHeight={variables.iconSizeNormal}
-                        wrapperStyle={styles.sectionMenuItem}
-                        focused={index === activeItemIndex}
-                        hoverAndPressStyle={styles.hoveredComponentBG}
-                        onPress={onPress}
-                        isPaneMenu
-                    />
-                );
-            })}
-        </View>
+        <>
+            <View style={[styles.pb4, styles.mh3, styles.mt3]}>
+                {typeMenuItems.map((item, index) => {
+                    const onPress = singleExecution(() => {
+                        SearchActions.clearAllFilters();
+                        Navigation.navigate(item.route);
+                    });
+
+                    return (
+                        <MenuItem
+                            key={item.title}
+                            disabled={false}
+                            interactive
+                            title={item.title}
+                            icon={item.icon}
+                            iconWidth={variables.iconSizeNormal}
+                            iconHeight={variables.iconSizeNormal}
+                            wrapperStyle={styles.sectionMenuItem}
+                            focused={index === activeItemIndex}
+                            hoverAndPressStyle={styles.hoveredComponentBG}
+                            onPress={onPress}
+                            isPaneMenu
+                        />
+                    );
+                })}
+            </View>
+            <ScrollView>{savedSearches && savedSearches.length > 0 && renderSavedSearchesSection(savedSearchesMenuItems())}</ScrollView>
```

Comment:
```suggestion
            <ScrollView>{savedSearches && Object.keys(savedSearches).length > 0 && renderSavedSearchesSection(savedSearchesMenuItems())}</ScrollView>
```
`savedSearches` is an object so it doesn't have a `length` property. Should resolve [this issue](https://github.com/Expensify/App/pull/48566#issuecomment-2348381644)!

Reply:
Fixed!